toolchain/check/import_ref.cpp

```diff
@@ -676,6 +676,69 @@ static auto AddImportIRInst(ImportContext& context, SemIR::InstId inst_id)
       {.ir_id = context.import_ir_id(), .inst_id = inst_id});
 }
 
+// Sets the constant value for an instruction and returns it. The constant value
+// is computed if not explicitly specified.
+static auto SetConstantValue(Context& context, SemIR::InstId inst_id,
+                             SemIR::Inst inst) -> SemIR::ConstantId {
+  auto const_id = TryEvalInstUnsafe(context, inst_id, inst);
+  if (const_id.has_value()) {
+    CARBON_VLOG_TO(context.vlog_stream(), "Constant: {0} -> {1}\n", inst,
+                   context.constant_values().GetInstId(const_id));
+  }
```

Comment:
Is this ever going to be `None`?
```suggestion
  CARBON_VLOG_TO(context.vlog_stream(), "Constant: {0} -> {1}\n", inst,
                 context.constant_values().GetInstId(const_id));
```